docs/installation/web-interface.md

```diff
@@ -8,11 +8,23 @@ nav_order: 2
 
 # Move2Kube Web Interface
 
+## Bringing up Move2Kube all-in-one container
+
+   ```console
+   $ mkdir -p workspace && cd workspace
+   $ docker run -p 8080:8080 -v $PWD:/workspace -v /var/run/docker.sock:/var/run/docker.sock -it quay.io/konveyor/move2kube-aio:latest
```

Comment:
I am able to run the all-in-one container successfully and access the UI at `http://localhost:8080/` using these two commands. Then, I tried translating the cloud foundry sample usecase. After the translation was completed, when I tried to download the generated target artifacts, it downloaded a single json file. Tried doing the translation for the language platforms sample and for that also a json file was downloaded when I clicked on `Get`.

<img width="827" alt="Screenshot 2020-12-28 at 7 49 37 PM" src="https://user-images.githubusercontent.com/14867323/103220361-15f75080-4946-11eb-9ab6-edbbf3aeafec.png">

Reply:
Can you try with chrome once? And also in the current one can you check the url pointed to by get?

Reply:
Yes, I tried with chrome it is working fine and I am able to download the zip file containing the generated artifacts. Hovering over the `Get` does not show the url, in the browser console the filename is `/api/v1/applications/newapp/targetartifacts/newapp_1609164843`.

<img width="549" alt="Screenshot 2020-12-28 at 8 15 16 PM" src="https://user-images.githubusercontent.com/14867323/103221766-68863c00-4949-11eb-8257-a97c5e0a39fc.png">

Reply:
Then it's some firefox compatibility issue. Let's merge this and create a new issue in move2kube-ui? Btw, did the docker-compose method work in Firefox?

Reply:
Yes sure. I just checked with docker-compose method also, I am facing the same issue with that as well.